zipkin-common/src/test/scala/com/twitter/zipkin/json/JsonConversionTest.scala

```diff
@@ -7,6 +7,19 @@ import com.twitter.zipkin.common.{Endpoint, AnnotationType, BinaryAnnotation}
 import org.scalatest.{FunSuite, Matchers}
 
 class JsonConversionTest extends FunSuite with Matchers {
+
+  test("span with null annotations, binaryAnnotations") {
```

Comment:
it is easier to completely replace the codec system than properly work around this in the version of jackson we are using.